ambry-store/src/test/java/com.github.ambry.store/StoreMessageReadSetTest.java

```diff
@@ -262,8 +262,8 @@ public void blobReadOptionsTest() throws IOException {
 
         try {
           new BlobReadOptions(log, new Offset(firstSegment.getName(), firstSegment.getEndOffset()),
-              new MessageInfo(null, 1, 1, Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID,
-                  Utils.Infinite_Time));
+              new MessageInfo(null, 1, 1, Utils.getRandomShort(TestUtils.RANDOM),
+                  Utils.getRandomShort(TestUtils.RANDOM), Utils.getRandomShort(TestUtils.RANDOM)));
```

Comment:
Why is this time random as opposed to `System.currentTimeMillis()`?

Reply:
These are unit test. For tools, we would go with System.currentTimeMillis()